Games/types/Mafia/roles/cards/GainKnifeIfVisited.js

```diff
@@ -0,0 +1,30 @@
+const Card = require("../../Card");
```

Comment:
This card should explicitly say "gain knife if visited by non monster", to avoid confusion with a town role that might have the same mechanic